src/core_modules/capture-core/rules/actionsCreator/runRulesForTEI.js

```diff
@@ -1,17 +1,17 @@
 // @flow
 import log from 'loglevel';
-import { errorCreator } from '../../capture-core-utils';
-import { RulesEngine } from '../../capture-core-utils/RulesEngine';
-import { TrackerProgram, DataElement, RenderFoundation } from '../metaData';
-import constantsStore from '../metaDataMemoryStores/constants/constants.store';
-import optionSetsStore from '../metaDataMemoryStores/optionSets/optionSets.store';
+import { errorCreator } from '../../../capture-core-utils';
+import { RulesEngine } from '../engine';
+import { TrackerProgram, DataElement, RenderFoundation } from '../../metaData';
+import constantsStore from '../../metaDataMemoryStores/constants/constants.store';
+import optionSetsStore from '../../metaDataMemoryStores/optionSets/optionSets.store';
 
 import type {
     ProgramRulesContainer,
     TrackedEntityAttribute as TrackedEntityAttributeForRulesEngine,
     Enrollment,
     TEIValues,
-} from '../../capture-core-utils/RulesEngine/rulesEngine.types';
+} from '../engine/rulesEngine.types';
```

Comment:
lets discuss about those, since it is not clear to me if these imports are from the same module or form a different module.

To me it looks the whole `rules` is its own module. What do you think?